common/src/main/resources/assets/wynntils/lang/zh_cn.json

```diff
@@ -686,6 +686,8 @@
   "feature.wynntils.worldWaypointDistance.textShadow.name": "文本阴影",
   "feature.wynntils.worldWaypointDistance.topBoundingDistance.description": "离屏幕时，航路点指示器最近对屏幕的顶部距离是多少？",
   "feature.wynntils.worldWaypointDistance.topBoundingDistance.name": "顶部提示器距离",
+  "feature.wynntils.wynncraftButton.autoConnect.description": "translate: Should you auto-connect to Wynncraft on startup?",
```

Comment:
Also, I am not sure we need these. I believe most features are merged with only an entry for en_us.json.

Reply:
Remove these, only keep en-us